frame/fast-unstake/src/lib.rs

```diff
@@ -177,6 +181,8 @@ pub mod pallet {
 		NotQueued,
 		/// The provided un-staker is already in Head, and cannot deregister.
 		AlreadyHead,
+		/// The call is not allowed at this point.
+		CallNotAllowed,
```

Comment:
People are going to ask why call not allowed. Can we have CallNotAllowedBecauseXyz or something more specific. The more specific error enum variants the better - means less support.

Reply:
In this case `CallNotAllowed` because the pallet is disabled (via setting eras to check to 0). 
I don't think we can be any more specific in this case tbh, this can happen due to Internal errors or just governance changes.
It also seems to be consistent with other pallets, but indeed not very informative.

Reply:
reasonable to rename I presume.

Reply:
The docs are shown in the UIs as well, so we only need to change the doc.